assets/app/views/deployments.html

```diff
@@ -10,14 +10,112 @@
-        <div>Status: {{deployment.metadata.annotations.deploymentStatus}}</div>
-        <div>Version: {{deployment.metadata.annotations.deploymentVersion}}</div>
-        <div>Replicas: {{deployment.spec.replicas}}</div>
-        <pod-template ng-init="podTemplate = deployment.spec.template"></pod-template>
+      <div class="tile" ng-repeat="(deploymentConfigName, deploymentConfig) in deploymentConfigs">
+        <div> 
+          <h2>{{deploymentConfigName}}</h2>
+          <dl class="dl-horizontal left indent">
+          <div>
+              <dt>Created:</dt>
+              <dd><relative-timestamp timestamp="deploymentConfig.metadata.creationTimestamp"></relative-timestamp></dd>
+          </div>
+          <div>
+              <dt>Labels:</dt>
+              <dd>{{deploymentConfig.metadata.labels}}</dd>
+          </div>
+          <div>
+              <dt>Latest Version:</dt>
+              <dd>{{deploymentConfig.latestVersion}}</dd>
+          </div>
+          <div>
+              <dt>Strategy:</dt>
+              <dd>{{deploymentConfig.template.strategy.type}}</dd>
+          </div>
+
+          <div>
+            <h3>Triggers:</h3>
+            <dl class="dl-horizontal left indent">
+              <div ng-repeat="trigger in deploymentConfig.triggers">
+                <span ng-switch="trigger.type">
+                  <span ng-switch-when="ImageChange">
+                    TODO: ImageChange                    
+                  </span>
+                  <span ng-switch-when="ConfigChange">                    
+                    TODO: ConfigChange                    
+                  </span>
+                  <span ng-switch-default>{{trigger.type}}</span>
+                </span>
+              </div>
+            </dl>
+
+            <dl class="dl-horizontal left indent">
+              <dt>Manual:</dt>
+              <dd>
+                <span>
+                  <button class="btn btn-primary" ng-click="" ng-disabled="">Start Deployment</button>
+                <span>
+              </dd>
+            </dl>
+
+            <dl class="dl-horizontal left indent">
+              <dt>Manual (CLI):
+                <a href="{{'start-deployment' | helpLink}}" target="_blank">
+                  <span class="learn-more-block">Learn more <i class="fa fa-external-link"> </i></span>
+                </a>
+              </dt>
+              <dd>
+                <code>osc deploy {{deploymentConfigName}} -n {{project.metadata.name}}</code>
+                <copy-to-clipboard-button clipboard-text="'osc deploy ' + deploymentConfigName + ' -n ' + project.metadata.name"></copy-to-clipboard-button>
+              </dd>
+
+            </dl>
+          </div>
+          
+          <div class="well" style="margin-bottom: 10px;" ng-repeat="deployment in deploymentsByDeploymentConfig[deploymentConfigName] | orderObjectsByDate : true">
```

Comment:
Seems like we'd want to collapse the old deployments that are complete and scaled down to 0 in a "View previous deployments..." toggle or something